crates/ruff/src/checkers/ctx.rs

```diff
@@ -0,0 +1,252 @@
+use nohash_hasher::IntMap;
```

Comment:
Nit: Should the file be named `context`? Avoiding abbreviations can improve readability for people less familiar with the code base.